src/Chirp.Web/Pages/Public.cshtml.cs

```diff
@@ -34,17 +34,9 @@ public PublicModel(ICheepService service, ICheepRepository cheeprepository, IAut
     }
 
     public ActionResult OnGet()
-    { 
-        int page;
-        if(Request.Query.ContainsKey("page")){
-            page = int.Parse(Request.Query["page"]! );
-        } else{
-            page = 1;
-        }
-        Cheeps = _service.GetCheeps(page);
+    {
```

Comment:
What's going on here? If it's not a mistake just ignore this comment, I just don't understand the syntax.